examples/prometheus/prometheus.yaml

```diff
@@ -22,6 +22,9 @@ parameters:
   name: SESSION_SECRET
   generate: expression
   from: "[a-zA-Z0-9]{43}"
+- description: Time interval for scrape config (1m, 5s)
+  name: SCRAPE_INTERVAL
+  value: 15s
```

Comment:
Let's not change the existing default of 1m.  This new variable allows us to override 1m (which we may do in all cases, but still, we do not want to change the default of 1m).